front/app/feed.xml/route.ts

```diff
@@ -1,8 +1,17 @@
 import { PAGE_URL } from "@/constants";
-import { useComicMetadata } from "@/hooks";
-import { getAllMetadata, getImageUrl } from "@/utils";
+import {Comic, getAllMetadata, getImageUrl, Metadata} from "@/utils";
 import RSS from "rss";
 
+function compare(a:{comic: Comic, series: Metadata}, b:{comic: Comic, series: Metadata}) {
+  if ( a.comic.uploaded_at < b.comic.uploaded_at ){
+    return 1;
+  }
+  if ( a.comic.uploaded_at > b.comic.uploaded_at ){
+    return -1;
+  }
+  return 0;
+}
```

Comment:
```suggestion
function compare(a: {comic: Comic}, b: {comic: Comic}) {
  const aDate = new Date(a)
  const bDate = new Date(b)
  return a.valueOf() - b.valueOf()
}
```

Fun fact, we don't really need to do this whole if dance, we can just return the result of subtraction

Reply:
was that meant to be `aDate.valueOf() - bDate.valueOf()`

Reply:
ok, assuming it was meant to be that, i have applied this change. had to adjust it for the combined type and inverted the subtraction to get newest first but it works and looks a lot cleaner